### DISCUSSION TITLE
Why does Github support unconstitutional activities?

### DISCUSSION THREAD

**Discussion by Songzen** [community/community#31866] [Other Feature Feedback, Questions, & Ideas]
Why does Github support community group harrassment of  ‘Special Surveillance Groups’ AKA gangstalking, covert harrassment, group harrassment, etc of non-investigative subjects?

There are a huge number of projects here that are oriented towards monitoring, hacking into, and covertly altering individuals devices software and hardware functioning as well as dead drops of information that can be used to access individuals accounts, decrypt ciphers crypted materials with individuals PII in them, as well as projects whose purpose is to stream materials from individuals devices to multiple devices in a manner that is unconstitutional and illegal. 

Are there no safety mechanisms in place here? Does anything go?

**Comment by jfullstackdev** [comment]
you have the report button, go to the user or Org page 

if it's a user, it's on the left side under the profile pic, if it's an org, it's on the right side under the public members

**Reply by FlorianWendelborn** [reply]
I only ever reported malware, but my experience so far is that this process is incredibly slow. GitHub didn’t even acknowledge that they’re looking into the report of the Trojan I found for 6 months. Around 1-2 months after acknowledging it, it’s still up and they still didn’t conclude their investigation.

I’ve left detailed instructions on how to find the malicious code and the impact it would have on unsuspecting users.

So there certainly needs to be some improvements in that part of the process as well. It’s just not fast enough if it takes like a year to remove repos like this.

**Reply by Songzen** [reply]
It's too widespread. It'd be like reporting something at scribd or pastebin

**Comment by FlorianWendelborn** [comment]
> in a manner that is unconstitutional and illegal

Not saying that nothing should be done, but please note that there is no World Constitution just yet. The USA is not the only country that exists, contrary to popular belief.

**Reply by Songzen** [reply]
I was just going by it being based in CA. Which I once heard wants to secede from the US and has an economy that is equivalent to its own nation. In any event I would then rephrase the question/assertion/etc to point out Geneva conventions, Nuremberg, etc. though technically that requires a nation state to be at war or something something yet there is clearly some fascist war going on against a subsection of the population (not going to write a thesis on this assertion, do your own research: to anyone who would ask me to back that up, I don't have time, I did my research and convinced myself). Also doesn't mean every nation state has agreed to those "guidelines" which are clearly disregarded when its convenient anyways. So when it comes down to it I'm maybe just asking why does Github support a huge percentage of projects that are aimed at illegally (in many countries according to their stated protections) extracting data from devices irregardless of the users acquiescence or knowledge thereof, delivering data to 3rd party corporations, setting up video and audio mirroring from said devices which is oftentimes streamed to multiple individuals and business entities, collecting data when the device is not connecting to the internet or any other form of local network (supposedly lol), and uploading that data elsewhere, allowing unknown entities to use spoofing and screen overlays and divert individuals browsers to fake sites, or otherwise change scripts to make buttons nonfunctional, change browser results, etc. I could go on but I think the point should be clearer now
